src/main/webapp/app/config/constants.tsx

```diff
@@ -34,6 +34,7 @@ export const APP_TIMESTAMP_FORMAT = 'MM/DD/YY HH:mm:ss';
 export const APP_LOCAL_DATE_FORMAT = 'MM/DD/YYYY';
 export const APP_LOCAL_DATETIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
 export const APP_LOCAL_DATETIME_FORMAT_Z = 'YYYY-MM-DDTHH:mm:ssZ';
+export const APP_LOCAL_DATETIME_FORMAT_Z_FORCE = 'YYYY-MM-DDTHH:MM:SS[Z]';
```

Comment:
why don't we use `APP_LOCAL_DATETIME_FORMAT_Z`?

Reply:
If we used `moment.add()` then `format(APP_LOCAL_DATETIME_FORMAT_Z)` it will not get this format but `YYYY-MM-DDTHH:mm:ss-H:00`. Here need to `[Z] `to convert the time zone `-H:00` on the tail.